lib/utilities/serverAccessLogger.js

```diff
@@ -0,0 +1,332 @@
+const { Werelogs } = require('werelogs');
+const { config } = require('../Config');
+const fs = require('fs');
+const path = require('path');
+const logger = require('./logger');
+
+const DEFAULT_OUTPUT_FILE = '/logs/api-operations.log';
+const SERVER_ACCESS_LOG_FORMAT_VERSION = '0';
+
+function createServerAccessLogger() {
+    if (!config.serverAccessLogs || !config.serverAccessLogs.enabled) {
+        logger.warn('ServerAccessLogs disabled returning no-op logger');
+        return {
+            info: () => { },
+            debug: () => { },
+            warn: () => { },
+            error: () => { },
+            trace: () => { },
+            fatal: () => { },
+        };
+    }
+
+    // Ensure logs directory exists
+    const outputFile = config.serverAccessLogs.outputFile || DEFAULT_OUTPUT_FILE;
+    const logDir = path.dirname(outputFile);
+
+    try {
+        if (!fs.existsSync(logDir)) {
+            fs.mkdirSync(logDir, { recursive: true });
+        }
+    } catch (error) {
+        // Fall back to logger-only logging if directory creation fails
+        logger.warn('Failed to create ServerAccess log directory, falling back to console logging:', error.message);
```

Comment:
why having this fallback option? Don't the logs get mixed with application logs?